CHANGELOG_en-GB.md

```diff
@@ -1,3 +1,11 @@
+# 3.0.0
+* Be aware, that this version only supports the shop versions starting from v6.5.4
```

Comment:
Is this composer locked?

Reply:
Yes, the composer entry is set during the packaging process within the workflow.